assets/js/base/utils/tracks.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { select } from '@wordpress/data';
+
+function isTracksAvailable() {
+	return typeof window.wcTracks.recordEvent === 'function';
```

Comment:
Just to check, is this only available after opt-in?

Reply:
Good catch – if store has opted out, this is still a function, just one that does nothing. Ideally we have a way to inhibit any tracks-related setup code from running when store has opted out.

I'll need to revisit this - this if is not working as intended right now.

Reply:
I've revised this (c7400b8af1a0a8e922032d47ad95a779cdeae9f9), using similar approach as core - `recordEvent` is a no-op, no mechanism to inhibit setup code related to tracks events (as far as I'm aware). 

If (in future) we find we need this we could look at adding info about tracks availability to wcSettings.